src/foam/graphics/TreeGraph.js

```diff
@@ -38,7 +38,7 @@
 
    methods: [
      function initCView() {
-       this.SUPER();
+      this.SUPER();
```

Comment:
Looks like indentation got thrown off here.

Reply:
Yeah but this space know aligns it up with the rest of the code.

Reply:
You sure about that? This looks like it's one off from the line below it. That said, you're right in that this whole file seems to be off by one but don't address that in this PR because it'll make reviewing it tough

Reply:
Done, you were off. there was a curly bracket that threw me off by aligning with the current indentation, so I just aligned those to the rest of the code.